android/app/src/main/java/com/gutenberg/Media.java

```diff
@@ -0,0 +1,65 @@
+package com.gutenberg;
+
+import androidx.annotation.NonNull;
+
+import org.wordpress.mobile.ReactNativeGutenbergBridge.GutenbergBridgeJS2Parent.MediaType;
+import org.wordpress.mobile.ReactNativeGutenbergBridge.GutenbergBridgeJS2Parent.RNMedia;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.bridge.WritableNativeMap;
+
+import java.util.Locale;
+
+public class Media implements RNMedia {
```

Comment:
I wish we had the kotlin dependency setup to reduce this to 4 LoCs :)

Reply:
https://github.com/wordpress-mobile/gutenberg-mobile/issues/2203 🤷‍♂️